docs/articles/configs/jobs.md

```diff
@@ -33,6 +33,9 @@ It's a single string characteristic. It allows to name your job. This name will
   * `CpuGroups`:  Specifies whether garbage collection supports multiple CPU groups
   * `Force`: Specifies whether the BenchmarkDotNet's benchmark runner forces full garbage collection after each benchmark invocation
   * `AllowVeryLargeObjects`:  On 64-bit platforms, enables arrays that are greater than 2 gigabytes (GB) in total size
+* `LargeAddressAware`: Specifies that BenchmarkDotNet's benchmark runner can handle addresses larger than 2 gigabytes. See also: @BenchmarkDotNet.Samples.IntroLargeAddressAware
+  * `false`: Benchmark runner uses the default (64-bit: enabled; 32-bit:disabled).
+  * `true`: Explicitly specify that BenchmarkDotNet's benchmark runner can handle addresses larger than 2 gigabytes.
```

Comment:
since this is BenchmarkDotNet docs, there is no need to specify project name or the runner.

```suggestion
* `LargeAddressAware`: Specifies that benchmark can handle addresses larger than 2 gigabytes. See also: @BenchmarkDotNet.Samples.IntroLargeAddressAware and [LARGEADDRESSAWARE](https://learn.microsoft.com/cpp/build/reference/largeaddressaware-handle-large-addresses)
  * `false`: Benchmark uses the defaults (64-bit: enabled; 32-bit: disabled).
  * `true`: Explicitly specify that benchmark can handle addresses larger than 2 gigabytes.
```

Please apply similar changes to the xml doc comments